src/Expectation.php

```diff
@@ -339,4 +340,9 @@ public static function hasMethod(string $name): bool
             || method_exists(Mixins\Expectation::class, $name)
             || self::hasExtend($name);
     }
+
+    public function any(): Any
```

Comment:
Added a description, and mark this method as internal for now.

Reply:
done